metadata/src/test/java/org/apache/kafka/metadata/placement/StripedReplicaPlacerTest.java

```diff
@@ -101,7 +102,7 @@ public Iterator<UsableBroker> usableBrokers() {
                 return brokers.iterator();
             }
         };
-        return placer.place(placementSpec, cluster);
+        return placer.place(placementSpec, cluster).assignments().stream().map(x -> x.replicas()).collect(Collectors.toList());
```

Comment:
Did you consider changing the signature of this method to return `TopicPlacement` and adjusting the test above to use that type?

Reply:
I was originally trying to reduce the number of changes. But I think it makes sense to return a TopicAssignment so we dont have to keep creating another list.